website/docs/user-guides/upgrade.md

```diff
@@ -0,0 +1,32 @@
+---
+id: upgrade
+title: Upgrade Litmus ChaosCenter to Latest Version
+sidebar_label: Upgrade
+---
+
+---
+
+## Upgrade ChaosCenter
+
+To upgrade Litmus ChaosCenter from 2.0.0-Beta9 to 2.0.0, you can follow these steps:
+
+1.  ## Control Plane Upgrade
+    To upgrade ChaosCenter with this approach, you can re-apply the 2.0.0 manifest: 
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/2.0.0/docs/2.0.0/litmus-2.0.0.yaml
+    ```
+
+2.  ## Agent Plane Upgrade
+    With this approach, all the ChaosAgents connected with ChaosCenter should be upgraded manually with [Litmusctl](../litmusctl/installation).
+
+    To do this, `disconnect` the ChaosAgent from ChaosCenter.
```

Comment:
@amityt thinking if we can add the disconnect + deploy cleanup on target cluster as a pre-requisite before entering the upgrade process? Rather than upgrade control plane and then do the disconnect?

Reply:
Also a note that asks to give same inputs in litmusctl connect operation would be helpful -- as that will ensure no duplicate agent components (workflow controller, operator et al) are there, and that there is an in-place upgrade